explainaboard/loaders/file_loader.py

```diff
@@ -547,6 +557,14 @@ def load_raw(
 
         # Infer metadata from the dataset
         metadata = FileLoaderMetadata()
+        # load customized features from global config files
+        if config.dataset in customized_features_from_config.keys():
```

Comment:
```suggestion
        if config.dataset in customized_features_from_config:
```

Reply:
I think `if config.dataset in customized_features_from_config.keys()` is correct here.

Reply:
Sorry, I wasn't clear enough on this. I think they're both functionally identical, but:
1. writing `.keys()` is more verbose, and
2. I believe `if x in y.keys()` is `O(len(y))`, where `if x in y` is `O(1)`, as the former returns a view of the keys as a list: https://www.w3schools.com/python/ref_dictionary_keys.asp

So I'd suggest not writing `.keys()`

Reply:
Actually I was wondering about "2." so I researched a bit more, and it seems that starting in Python 3.x both are `O(1)`: https://stackoverflow.com/a/17539423

But still, not writing `.keys()` is less verbose, so I'd recommend it either way.

Reply:
oh, I see. Thanks for the patient explanation!